addons/config/expected/cpi.tanzu.vmware.com/v1alpha1/vspherecpiconfig.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: cpi.tanzu.vmware.com/v1alpha1
+kind: VSphereCPIConfig
+metadata:
+  name: v1.23.3---vmware.1-tkg.1
+  namespace: tkg-system
+spec:
+  vsphereCPI:
+    mode: vsphereCPI
```

Comment:
Can you add testcase for other iaas mode too. Thanks

Reply:
yes, I have added a test for paravirt mode. could you check the `addons/config/hack/test.sh`?